docs/agents/keithley-psu.rst

```diff
@@ -1,4 +1,94 @@
-==============================
-Keithly Power Supplies [empty]
-==============================
+.. highlight:: rst
 
+.. _keithly_psu:
+
+==================
+Keithly PSU Agent
+==================
+
+This agent works for many power supplies, including the Keithly 2230G 
+and BK Precision 9130. It connects to the PSU over ethernet, and allows
```

Comment:
If this isn't Keithly specific, is there a better, more general name we can rename this Agent to? Perhaps something related to the protocol/interface used to communicate with the PSUs?

Reply:
Yeah this is probably a good time to do that: both power supplies run off of the SCPI (standard commands for programmable interfaces) formalism, which is why the agent works for both. How about something like psu_scpi? I'll wait to make the change everywhere until we agree on a name.

Reply:
That name sounds good, but I'd prefer we flip the words, so scpi_psu, describing the protocol first, then a description of the device it works for.

Reply:
Just did in the latest commit